tests/wp-unit/include/Core/Terms/Modules/DeleteTermsByPostCountModuleTest.php

```diff
@@ -0,0 +1,202 @@
+<?php
```

Comment:
@rajanit2000, 

There are 15 Code sniffer errors on this file too. Could you get it fixed?